src/PhpBrew/Build.php

```diff
@@ -98,6 +98,19 @@ public function setOSRelease($osRelease)
         $this->osRelease = $osRelease;
     }
 
+    public function setPkgConfigPath($path)
```

Comment:
Since this method effectively collects the paths, let's redefine it as:
```php
/** @var array<string,true> */
private $pkgConfigPaths = array();

public function getPkgConfigPaths()
{
    return array_keys($this->pkgConfigPaths);
}

public function addPkgConfigPath($path)
{
    $this->pkgConfigPaths[$paths] = true;
}

```

This way, the paths will be collected and naturally duplicated, and then whatever component should use these paths, could call `$build->getPkgConfigPaths()` and pass them to the `configure` command. This way, instead of just setting the environment variable globally, it would be better manageable and testable.

Reply:
I can do this, but I need another method to put them to shell environment, for example `setPkgConfigPaths()`. Since we don't have `Environment` class, maybe we can add it to `Build` class, `Config` class, or `VariantBuilder` itself. I am not sure which one is better.

And, I can call it after we build options from `$option=$variantBuilder->build($build)`.

The pseudo code may looks like: 

```php
       // InstallCommand.php 
       // add method setPkgConfigPaths() inside Build class, call it after VariantBuilder build $option
        $variantBuilder = new VariantBuilder();
        $configureOptions = $variantBuilder->build($build);
        $build->setPkgConfigPaths());
```
or
```php
        // InstallCommand.php 
        // add method putPkgConfigPaths() inside Config class, call it after VariantBuilder build $option
        $variantBuilder = new VariantBuilder();
        $configureOptions = $variantBuilder->build($build);
        Config::setPkgConfigPaths($build->getPkgConfigPaths());
```

Reply:
Fixed!